tidb-cloud/backup-and-restore.md

```diff
@@ -8,27 +8,65 @@ aliases: ['/tidbcloud/restore-deleted-tidb-cluster']
 
 This document describes how to back up and restore your TiDB cluster data on TiDB Cloud.
 
-> **Note:**
->
-> For [Serverless Tier clusters](/tidb-cloud/select-cluster-tier.md#serverless-tier-beta), the backup and restore feature is unavailable. You can use [Dumpling](https://docs.pingcap.com/tidb/stable/dumpling-overview) to export your data as a backup.
->
-> Currently, the backup and restore feature has the following restrictions:
->
-> - TiDB Cloud does not support restoring tables in the `mysql` schema, including user permissions and system variables. You can use [Dumpling and Lightning](https://docs.pingcap.com/tidb/dev/backup-and-restore-using-dumpling-lightning) to manually back up and restore these data.
-> - If you turn on and off PITR (Point-in-time Recovery) multiple times, you can only choose a time point within the recoverable range after the most recent PITR is enabled. The earlier recoverable range is not accessible.
+## Limitations
+
+### Dedicated Tier
+- TiDB Cloud does not support restoring tables in the `mysql` schema, including user permissions and system variables. You can use [Dumpling and Lightning](https://docs.pingcap.com/tidb/dev/backup-and-restore-using-dumpling-lightning) to manually back up and restore these data.
+- If you turn on and off PITR (Point-in-time Recovery) multiple times, you can only choose a time point within the recoverable range after the most recent PITR is enabled. The earlier recoverable range is not accessible.
+
+### Serverless Tier
+- It is important to note that Serverless Tier clusters only support in-place restoring from backups. When a restore is performed, tables in the `mysql` schema are also impacted, hence any changes made in user credentials and permissions or system variables will be rolled back to the state when the backup was taken.
+- Manual backup is not yet supported at this time.
+- PITR(Point-in-time Recovery) is not yet supported at this time.
+- The cluster will be unavailable during the restore process and existing connection will be cutoff. You can establish new connections once the restore is complete.
+- If TiFlash replica is enabled, it will be unavailable for a while after restore because data needs to be rebuilt.
```

Comment:
```suggestion
- The cluster will be unavailable during the restore process, and existing connections will be terminated. You can establish new connections once the restore is complete.
- If any TiFlash replica is enabled, the replica will be unavailable for a while after the restore because data needs to be rebuilt in TiFlash.
```